change/@fluentui-react-components-ea4050e0-fb62-4709-a678-0baf10d7bcfa.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add Input to react-components",
```

```suggestion
  "comment": "Add Input components (exported as /unstable)",
```

As this goes to the changelog, let's improve it a bit